ci/requirements.txt

```diff
@@ -0,0 +1,18 @@
+dateutil
+pytz
+openpyxl
+xlsxwriter
+xlrd
+html5lib
+patsy
+beautiful-soup
+numpy
+cython
+scipy
+numexpr
+pytables
+matplotlib
+lxml
+sqlalchemy
+bottleneck
+pymysql
```

Comment:
i left out ``psycopg2``, for some reason could not get this install correctly on py3.4 (it works if you install it after though). weird.